bioblend/galaxy/tool_dependencies/__init__.py

```
@@ -0,0 +1,82 @@
+
+        :type index_by: str
+        :param index_by: By default results are grouped by requirements.  Set to 'tool'
+            to return one entry per tool.
+
+        :rtype: list of dicts
+        :returns: dictified descriptions of the dependencies, with attribute
+            `dependency_type: None` if no match was found.
+            For example::
+
+            [{'requirements': [{'name': 'galaxy_sequence_utils',
+                                'specs': [],
+                                'type': 'package',
+                                'version': '1.1.4'},
+                               {'name': 'bx-python',
+                                'specs': [],
+                                'type': 'package',
+                                'version': '0.8.6'}],
+              'status': [{'cacheable': False,
+                          'dependency_type': None,
+                          'exact': True,
+                          'model_class': 'NullDependency',
+                          'name': 'galaxy_sequence_utils',
+                          'version': '1.1.4'},
+                          {'cacheable': False,
+                          'dependency_type': None,
+                          'exact': True,
+                          'model_class': 'NullDependency',
+                          'name': 'bx-python',
+                          'version': '0.8.6'}],
+              'tool_ids': ['vcf_to_maf_customtrack1']}]
+        """
+
+        params = {}
+        if index:
+            params['index'] = str(index)
+        if tool_ids:
+            params['tool_ids'] = ','.join(tool_ids)
+        if resolver_type:
+            params['resolver_type'] = resolver_type
+        if include_containers is not None:
+            params['include_containers'] = str(include_containers)
+        if container_type:
+            params['container_type'] = container_type
+        if index_by:
+            params['index_by'] = index_by
```

Maybe throw an error if it's not one of tools or requirements?